source/options.html

```diff
@@ -41,6 +42,8 @@
 		<summary><strong>💅 Custom CSS</strong></summary>
 		<p>Like a userstyle, useful to undo unwanted style changes</p>
 		<p><textarea name="customCSS" rows="2" spellcheck="false"></textarea></p>
+		<p>CSS is limited to 100KB, after which the options <a href="https://github.com/fregante/webext-options-sync/issues/27">will stop being saved</a>.</p>
+		<p>Options storage: <output class="storage-sync">unknown</output></p>
```

Comment:
This is automatically updated on save, but if the user pastes 101KB nothing happens.

I added a link to the issue to help people find it and fix it